stories/documentation/navigation/vertical-navigation/vertical-navigation-basic.stories.ts

```diff
@@ -0,0 +1,54 @@
+import { Meta, Story } from '@storybook/angular';
+
+interface VerticalNavigationBasicStory {
+	disabled: boolean;
+}
+
+export default {
+	title: 'Documentation/Navigation/VerticalNavigation/Basic',
+	argTypes: {
+		disabled: {
+			control: {
+				type: 'boolean',
+			},
+		},
+	},
+} as Meta;
+
+function getTemplate(args: VerticalNavigationBasicStory): string {
+	const disabled = args.disabled ? `disabled tabindex="-1"` : '';
+	return `<nav class="verticalNavigation">
+	<ul class="verticalNavigation-list">
+		<li class="verticalNavigation-list-item" aria-expanded="false" aria-controls="childList1">
```

Comment:
According to the guidelines the `aria-expanded` attribute should be on the toggle trigger just before the item to expand or collapse so on the link or button below. The `aria-controls` attribute wouldn't be necessary in that case.
https://www.accede-web.com/en/guidelines/rich-interface-components/show-hide-panels/

Reply:
Thanks, fixed.